Remove docs from the repo (#479)
thrackle-io/forte-rules-engine: README.md

Reviewer: I feel like we should actually link to the docs at the top of the main README somewhere, e.g. right before "Installation". So people don't have to scroll down to find the docs.

[version-image]: https://img.shields.io/badge/Version-2.2.1-brightgreen?style=for-the-badge&logo=appveyor
[version-url]: https://github.com/thrackle-io/forte-rules-engine

There are two ways most projects may utilize the protocol. 
1. 

> 

Author: haha up, nice catch!

    3.  `python-dotenv 1.0.1`

    These packages can be installed manually or through the following helper command:

```bash
pip3 install -r requirements.txt
```

1. `@openzeppelin/contracts` version 4.9.6 and `@openzeppelin/contracts-upgradeable` version 4.9.6.

    If the contracts show any compiling errors, try to manually update the version of the existing openzeppelin library in your project by doing:

```bash
forge install OpenZeppelin/openzeppelin-contracts
```

```bash
forge install OpenZeppelin/openzeppelin-contracts-upgradeable
```

## Usage

### User Guides

> For complete usage information and documentation, please visit the docs at https://docs.forterulesengine.io.

Reviewer: ```suggestion
For complete usage information and documentation, please visit [our docs](https://docs.forterulesengine.io).
```
This might be contentious, so I'm ok leaving it as is, but I personally this looks more professional. Although I see you did it differently in the next section - is there a reason you presented the URL plainly here?

> ### Example Application

Reviewer: I'm not sure it's useful to have this "Example Application" section here, maybe we remove it given our move to the docs platform? Feels a little random in this context in my opinion.

Reviewer: Come to think of it - If we add a link at the top of the README to the docs, then maybe we don't need the "User Guide" section either here.
